asv/commands/show.py

```diff
@@ -206,22 +202,22 @@ def get_stat_info(key):
 
         for key in ['repeat', 'number', 'ci_99', 'mean', 'std', 'min', 'max']:
             values = get_stat_info(key)
-
             if key == 'ci_99':
-                values = ["({}, {})".format(util.human_value(x[0], benchmark['unit']),
-                                            util.human_value(x[1], benchmark['unit']))
+                benchmark_unit1 = util.human_value(x[0], benchmark['unit'])
+                benchmark_unit2 = util.human_value(x[1], benchmark['unit'])
+                values = [f"({benchmark_unit1},{benchmark_unit2}"
```

Comment:
This doesn't generate the same string, there are a space and a parenthesis missing.